vhost/src/vhost_user/frontend.rs

```diff
@@ -935,7 +932,7 @@ mod tests {
 
         let pfeatures = VhostUserProtocolFeatures::all();
         let hdr = VhostUserMsgHeader::new(FrontendReq::GET_PROTOCOL_FEATURES, 0x4, 8);
-        let msg = VhostUserU64::new(pfeatures.bits());
+        let msg = VhostUserU64::new(pfeatures.bits() | 1 << 63);
```

Comment:
nit: Maybe worth adding a comment to mention that we test that only supported features are returned (iiuc it is the reason for setting this bit) or something along those lines?

Other than that, lgtm :)